src/style.css

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+}
+
+body {
+  margin-top: 0;
+  font-weight: 300;
+}
+
+[data-qa*="big"] {
+  margin: 20px 0;
+}
+
+[data-qa*="big"] input {
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-position: 26px 25px;
+
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+
+  font-family: Avenir, sans-serif;

```

Comment:
move into `html`

Reply:
When I move font-family to html this property is not inherited for ::placeholder but with * it works

Reply:
It is not allowed to use rules besides box-sizing at `* {}`
If you need to apply current font to the `::placeholder` try to use `font-family: inherit`